### PR TITLE
update form new node

### PR DESCRIPTION
**Pull request by alavieille**
https://github.com/open-orchestra/open-orchestra-front-bundle/pull/208
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/694
https://github.com/open-orchestra/open-orchestra/pull/1100
https://github.com/open-orchestra/open-orchestra-model-interface/pull/261
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2009

### CODE REVIEW

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/Node/NodeRouter.js [hunk 1]
```diff
@@ -74,6 +76,57 @@ class NodeRouter extends OrchestraRouter
...
+    }
+
+    /**
+     *  New Node
+     *
+     * @param {string} language
+     * @param {string} parentId
+     * @param {int}    order
+     */
+    newNode(language, parentId, order) {
```
Comment (ngilain) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/Node/NodeRouter.js]: @alavieille : new is not an action, maybe createNode ?

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Node/NodeNewFormView.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import AbstractFormView from '../../../Service/Form/View/AbstractFormView'
+
+/**
+ * @class NodeNewFormView
+ */
+class NodeNewFormView extends AbstractFormView
```
Comment (ngilain) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Node/NodeNewFormView.js]: @alavieille : maybe NewNodeFormVIew instead ?

**Comment by alavieille** [comment]
@ngilain  pr updated i rename the method

**Comment by ngilain** [comment]
:+100:

**Pull request merged**